shell/components/ResourceDetail/Masthead.vue

```diff
@@ -435,7 +435,7 @@ export default {
         <div class="actions-container">
           <div class="actions">
             <button
-              v-if="detailsAction"
+              v-if="detailsAction && currentView === 'detail' && isView"
```

Comment:
Possibly controversial request, can we match the padding?

![image](https://user-images.githubusercontent.com/18697775/208167794-5ba6002e-030a-460d-a40c-1d5bd66e2346.png)

![image](https://user-images.githubusercontent.com/18697775/208167724-409e1698-1862-4613-b81f-2faaa25ecb60.png)

This would apply everywhere, but i think it looks alright?

![image](https://user-images.githubusercontent.com/18697775/208168068-4a546ed7-b1c1-44d3-a338-9293787f2fff.png)

Reply:
Done